example/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Why this change?

Reply:
Not required. I added a package.json to the example/ directory.

Reply:
Removed.